problems/sep_5_2018.js

```diff
@@ -1,11 +1,27 @@
-// Using the JavaScript language, have the function LetterChanges(str) take the str parameter being passed and modify it using the following algorithm:
+// Using the JavaScript language, have the function LetterChanges(str) take
+// the str parameter being passed and modify it using the following algorithm:
 
-// Replace every letter in the string with the letter following it in the alphabet (ie. c becomes d, z becomes a). Then capitalize every vowel in this new string (a, e, i, o, u) and finally return this modified string.
+// Replace every letter in the string with the letter following it
+// in the alphabet (ie. c becomes d, z becomes a).
+// Then capitalize every vowel in this new string (a, e, i, o, u)
+// and finally return this modified string.
+
+// Test case:
+// 'hello*3 => Ifmmp*3
 
 function LetterChanges(str) {
-  // code goes here
-  return str
+  const letterShift = str.replace(/[a-z]/gi, (char) => {
+    return char === 'z' || char === 'Z'
+      ? 'a'
+      : String.fromCharCode(char.charCodeAt() + 1)
```

Comment:
I got this from googling. I had no idea about the charCode methods. What I was doing was going nowhere.